assets/svgxml/iconSvgSelector.tsx

```diff
@@ -0,0 +1,27 @@
+</svg>
+`;
+
+const favorites = `<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12.9195 3.072C13.2928 2.08733 14.7057 2.08733 15.0802 3.072L17.4952 9.76166C17.6643 10.205 18.0948 10.5002 18.5755 10.5002H24.5103C25.607 10.5002 26.0853 11.8652 25.2232 12.5337L20.9998 16.3335C20.8107 16.4789 20.6724 16.6806 20.6049 16.9094C20.5374 17.1383 20.5442 17.3827 20.6242 17.6075L22.1665 24.1443C22.5422 25.1943 21.3265 26.0962 20.4072 25.4498L14.6707 21.8098C14.4742 21.6718 14.24 21.5977 13.9998 21.5977C13.7597 21.5977 13.5255 21.6718 13.329 21.8098L7.59251 25.4498C6.67434 26.0962 5.45751 25.1932 5.83318 24.1443L7.37551 17.6075C7.45552 17.3827 7.46226 17.1383 7.39477 16.9094C7.32728 16.6806 7.18903 16.4789 6.99984 16.3335L2.77651 12.5337C1.91318 11.8652 2.39385 10.5002 3.48818 10.5002H9.42301C9.65668 10.5009 9.88509 10.4307 10.078 10.2989C10.2709 10.167 10.4192 9.97967 10.5033 9.76166L12.9183 3.072H12.9195Z" stroke="#B6B6EA" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
+`;
+
+const profile = `<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M14 14C15.3924 14 16.7277 13.4469 17.7123 12.4623C18.6969 11.4777 19.25 10.1424 19.25 8.75C19.25 7.35761 18.6969 6.02226 17.7123 5.03769C16.7277 4.05312 15.3924 3.5 14 3.5C12.6076 3.5 11.2723 4.05312 10.2877 5.03769C9.30312 6.02226 8.75 7.35761 8.75 8.75C8.75 10.1424 9.30312 11.4777 10.2877 12.4623C11.2723 13.4469 12.6076 14 14 14ZM17.5 8.75C17.5 9.67826 17.1313 10.5685 16.4749 11.2249C15.8185 11.8813 14.9283 12.25 14 12.25C13.0717 12.25 12.1815 11.8813 11.5251 11.2249C10.8687 10.5685 10.5 9.67826 10.5 8.75C10.5 7.82174 10.8687 6.9315 11.5251 6.27513C12.1815 5.61875 13.0717 5.25 14 5.25C14.9283 5.25 15.8185 5.61875 16.4749 6.27513C17.1313 6.9315 17.5 7.82174 17.5 8.75ZM24.5 22.75C24.5 24.5 22.75 24.5 22.75 24.5H5.25C5.25 24.5 3.5 24.5 3.5 22.75C3.5 21 5.25 15.75 14 15.75C22.75 15.75 24.5 21 24.5 22.75ZM22.75 22.743C22.7482 22.3125 22.4805 21.0175 21.294 19.831C20.153 18.69 18.0057 17.5 14 17.5C9.9925 17.5 7.847 18.69 6.706 19.831C5.5195 21.0175 5.2535 22.3125 5.25 22.743H22.75Z" fill="#B6B6EA"/>
+</svg>
+`;
+
+
+const getIcon = (icon:string) => {
+    return (
+        <SvgXml xml={icon===Screen.HOME_PAGE ? home : icon===Screen.FAV_PAGE ? favorites : profile}/>
```

Comment:
1. nested ternary is something to avoid when given the chance, switch case is more readable and easier to maintain if you have to support new icons in the future. 
2. the getIcon is only readable to you as the developer, which icon am i getting? I think a better way is to have a each icon as function to be more readable such as const profileIcon, (i would also remove the get) 
and if the function does not receive any params you can use it as a component <ProfileIcon/>